src/Pos/Block/Arbitrary.hs

```diff
@@ -167,12 +157,13 @@ instance Arbitrary (SscPayloadDependsOnSlot ssc) =>
         return $ T.MainBody txPayload mpcData dlgPayload mpcUpload
 
 instance Arbitrary (SscPayload ssc) => Arbitrary (T.Body (T.MainBlockchain ssc)) where
-    arbitrary = makeSmall $ do
-        txPayload   <- arbitrary
-        mpcData     <- arbitrary
-        mpcProxySKs <- arbitrary
-        mpcUpload   <- arbitrary
-        return $ T.MainBody txPayload mpcData mpcProxySKs mpcUpload
+    arbitrary = makeSmall genericArbitrary
+    shrink mb = [ T.MainBody txp sscp dlgp updp
```

Comment:
🤦‍♂️ 
Thanks for spotting this. That's the Haddock page I visit the most, yet I forgot that.